Sources/TensorFlow/Optimizers/MomentumBased.swift

```diff
@@ -208,8 +208,8 @@ where
 ///
 /// - Parameters:
 ///   - learningRate: A Float. The learning rate (default value: 1e-3).
-///   - beta1: A Float. The exponentian decay rate for the 1st moment estimates (default value: 0.9).
-///   - beta2: A Float. The exponentian decay rate for the 2nd moment estimates (default value: 
+///   - beta1: A Float. The exponential decay rate for the 1st moment estimates (default value: 0.9).
```

Comment:
Thanks for the typo fix!

Actually, these doc comments have further room for improvement:
- No doc comments should repeat type information: remove `A Float`.
- The standard doc comments for default argument values is: ``The default value is `xxx`.``

```suggestion
///   - beta1: The exponential decay rate for the 1st moment estimates. The default value is `0.9`.
```

Reply:
@dan-zheng let me know what you think!